### PR TITLE
Support s02 Episodes

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Adds support for a short series project with 3 promos weeks and one week of radio program at the end.

### CODE REVIEW

File: configuration/coding_plans.py [hunk 1]
```diff
@@ -487,6 +487,87 @@ def get_rqa_coding_plans(pipeline_name):
...
+S02_RQA_CODING_PLAN = [
+    CodingPlan(raw_field="covid19_promo_s02e01_raw",
+               time_field="sent_on",
+               run_id_field="covid19_promo_s02e01_run_id",
+               coda_filename="IMAQAL_COVID19_promo_s02e01.json",
+               icr_filename="covid19_promo_s02e01.csv",
+               coding_configurations=[
+                   CodingConfiguration(
+                       coding_mode=CodingModes.MULTIPLE,
+                       code_scheme=CodeSchemes.S02E01_PROMO,
```
Comment (as2388) [inline configuration/coding_plans.py]: If the field name is promo_s02e01 then the code scheme should probably be called PROMO_S02E01

File: configuration/s01_pipeline_config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "PipelineName": "IMAQAL_COVID19",
+  "PipelineName": "imaqal_covid19_all_seasons",
```
Comment (as2388) [inline configuration/s01_pipeline_config.json]:
```suggestion
  "PipelineName": "IMAQAL_COVID19_all_seasons",
```
for consistency of formatting with Coda and the traffic dashboards

File: configuration/s02_pipeline_config.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  "DriveUpload": {
+    "DriveCredentialsFileURL": "gs://avf-credentials/pipeline-runner-service-acct-avf-data-core-64cc71459fe7.json",
+    "ProductionUploadPath": "imaqal_covid19_analysis_outputs/s02/production.csv",
+    "MessagesUploadPath": "imaqal_covid19_analysis_outputs/s02/messages.csv",
+    "IndividualsUploadPath": "imaqal_covid19_analysis_outputs/s02/individuals.csv",
+    "AutomatedAnalysisDir": "imaqal_covid19_analysis_outputs/s02/automated_analysis"
+  },
+  "MemoryProfileUploadBucket":"gs://avf-pipeline-logs-performance-nearline",
+  "DataArchiveUploadBucket": "gs://pipeline-execution-backup-archive",
+  "BucketDirPath": "2021/IMAQAL_COVID19/"
```
Comment (as2388) [inline configuration/s02_pipeline_config.json:96]: This might need to be 2020 given that's when this project started, not sure which makes more sense

**Review by as2388 — APPROVED** [review]

File: configuration/coding_plans.py (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/coding_plans.py]: updated thanks.

File: configuration/s01_pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/s01_pipeline_config.json]: updated thanks.

File: configuration/s02_pipeline_config.json (hunk 3 above)
Comment (IsaackMwenda) [reply configuration/s02_pipeline_config.json:96]: I think we use the year the series started instead of the project as we do on Wusc.

**Pull request merged**